roles/cluster_logging/files/kibana/provision-index.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+
+#
+# Script to setup the default index pattern in kibana
+#
+
+set -ue
+
+INDEX_ID="logstash-*"
+
+API_HOSTNAME="$KIBANA_SERVICE_HOST:$KIBANA_SERVICE_PORT"
```

Comment:
Which is the same service hardcoded differently (now the port is variable). I'm good with that for v1, but still something that need a ticket